src/test/java/com/aspose/imaging/cloud/test/api/UpdateImageApiTests.java

```diff
@@ -67,36 +62,8 @@ public void setUp() throws Exception {
      */
     @Test
 	@Parameters({
-		".bmp, true,", 
```

Comment:
@MaxCrank Why you remove these test cases?

Reply:
@roxboris please check corporate mail regarding tests across all levels.